datasets/mur.yaml

```diff
@@ -0,0 +1,60 @@
+Name: Multi-Scale Ultra High Resolution (MUR) Sea Surface Temperature (SST) on AWS
+Description: A GHRSST Level 4 gridded daily 1km merged sea surface temperature data set 
+Documentation: https://podaac.jpl.nasa.gov/dataset/MUR-JPL-L4-GLOB-v4.1
+Contact: podaac@podaac.jpl.nasa.gov
+ManagedBy: https://podaac.jpl.nasa.gov 
+UpdateFrequency: Daily
+Tags:
+  - aws-pds
```

Comment:
Please use tags in https://github.com/awslabs/open-data-registry/blob/master/tags.yaml.